Main-de.string

```diff
@@ -0,0 +1,717 @@
+
+/* Class = "NSMenuItem"; title = "Open Recent"; ObjectID = "tXI-mr-wws"; */
+"tXI-mr-wws.title" = "Öffne letzte";
+
+/* Class = "NSMenu"; title = "Kern"; ObjectID = "tlD-Oa-oAM"; */
+"tlD-Oa-oAM.title" = "Kern";
+
+/* Class = "NSTextFieldCell"; title = "25 %"; ObjectID = "uMr-IC-jPW"; */
+"uMr-IC-jPW.title" = "25 %";
+
+/* Class = "NSMenu"; title = "Presentify"; ObjectID = "uQy-DD-JDr"; */
+"uQy-DD-JDr.title" = "Presentify";
+
+/* Class = "NSMenuItem"; title = "Cut"; ObjectID = "uRl-iY-unG"; */
+"uRl-iY-unG.title" = "Ausschneiden";
+
+/* Class = "NSButtonCell"; title = "Round Rect"; ObjectID = "uwV-Fv-vsh"; */
+"uwV-Fv-vsh.title" = "Abgerundetes Rechteck";
+
+/* Class = "NSMenuItem"; title = "Paste Style"; ObjectID = "vKC-jM-MkH"; */
+"vKC-jM-MkH.title" = "Stil einfügen";
+
+/* Class = "NSMenuItem"; title = "Show Ruler"; ObjectID = "vLm-3I-IUL"; */
+"vLm-3I-IUL.title" = "Lineal anzeigen";
+
+/* Class = "NSButton"; ibShadowedToolTip = "Arrows"; ObjectID = "vMM-vc-8Oy"; */
+"vMM-vc-8Oy.ibShadowedToolTip" = "Pfeile";
+
+/* Class = "NSMenuItem"; title = "Clear Menu"; ObjectID = "vNY-rz-j42"; */
+"vNY-rz-j42.title" = "Menü leeren";
+
+/* Class = "NSButtonCell"; title = "Highlight with a full circle instead of a ring"; ObjectID = "vbl-Bg-qj4"; */
+"vbl-Bg-qj4.title" = "Mit einem vollen Kreis anstatt einem Ring markieren";
+
+/* Class = "NSTextFieldCell"; title = "1st Fav Color"; ObjectID = "vli-ph-UGB"; */
+"vli-ph-UGB.title" = "Erste favorisierte Farve";
+
+/* Class = "NSMenuItem"; title = "Make Upper Case"; ObjectID = "vmV-6d-7jI"; */
+"vmV-6d-7jI.title" = "Großgeschrieben";
+
+/* Class = "NSMenu"; title = "Ligatures"; ObjectID = "w0m-vy-SC9"; */
+"w0m-vy-SC9.title" = "Ligaturen";
+
+/* Class = "NSTextFieldCell"; title = "Free Hand"; ObjectID = "w8F-cR-EKY"; */
+"w8F-cR-EKY.title" = "Freihand";
+
+/* Class = "NSMenuItem"; title = "Align Right"; ObjectID = "wb2-vD-lq4"; */
+"wb2-vD-lq4.title" = "Rechtsbündig";
+
+/* Class = "NSMenuItem"; title = "Help"; ObjectID = "wpr-3q-Mcd"; */
+"wpr-3q-Mcd.title" = "Hilfe";
+
+/* Class = "NSMenuItem"; title = "Copy"; ObjectID = "x3v-GG-iWU"; */
+"x3v-GG-iWU.title" = "Kopieren";
+
+/* Class = "NSMenuItem"; title = "Use All"; ObjectID = "xQD-1f-W4t"; */
+"xQD-1f-W4t.title" = "Alle benutzen";
+
+/* Class = "NSMenuItem"; title = "Speech"; ObjectID = "xrE-MZ-jX0"; */
+"xrE-MZ-jX0.title" = "Sprache";
+
+/* Class = "NSWindow"; title = "Preferences"; ObjectID = "yMR-WQ-lTM"; */
+"yMR-WQ-lTM.title" = "Einstellungen";
+
+/* Class = "NSMenuItem"; title = "None"; ObjectID = "ypj-kv-Bcb"; */
+"ypj-kv-Bcb.title" = "Keine";
+
+/* Class = "NSMenuItem"; title = "Show Substitutions"; ObjectID = "z6F-FW-3nz"; */
+"z6F-FW-3nz.title" = "Ersetzungen anzeigen";
+
+/* Class = "NSTextFieldCell"; title = "Line Weight"; ObjectID = "zxt-da-umM"; */
+"zxt-da-umM.title" = "Linienbreite";
+
+/* Class = "NSTextFieldCell"; title = "Decrease Line Weight"; ObjectID = "rij-ew-yIo"; */
+"rij-ew-yIo.title" = "Linienbreite verringern";
```

Comment:
Here you translated the same string (Line weight) differently. I think "Liniendicke" is the better word.